core/snippets/snippets/commentBlock.snippet

```diff
@@ -0,0 +1,21 @@
+name: commentBlock
+phrase: block comment
+insertionScope: statement
+
+$0.insertionFormatter: CAPITALIZE_FIRST_WORD
+---
+
+language: c | javascript | java
+-
+/* $0 */
+---
+
+language: python
+-
+"""$0"""
```

Comment:
is this reall how you're supposed to do a comment block in python? I guess it works

Reply:
I don't know of any other way